src/util/TunnelIPv6Interface.h

```diff
@@ -64,8 +64,8 @@ class TunnelIPv6Interface : public nl::UnixSocket
 	bool add_address(const struct in6_addr *addr, int prefixlen = 64);
 	bool remove_address(const struct in6_addr *addr, int prefixlen = 64);
 
-	bool add_route(const struct in6_addr *route, int prefixlen = 64);
-	bool remove_route(const struct in6_addr *route, int prefixlen = 64);
+	bool add_route(const struct in6_addr *route, int prefixlen = 64, uint32_t metric = 512);
```

Comment:
What does 512 mean? Could this be replaced with a named constant?

Reply:
Actually I am leaning towards not having a default value in function declaration...(this way the caller need to be mindful when it uses this method)... Will change this.